dev-python/spyder-kernels/spyder-kernels-1.8.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Kernels used by spyder on its ipython console"
+HOMEPAGE="
+	https://github.com/spyder-ide/spyder-kernels/
+	https://pypi.org/project/spyder-kernels/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/wurlitzer[${PYTHON_USEDEP}]
+		>=dev-python/ipykernel-5.1.3[${PYTHON_USEDEP}]
+		dev-python/pyzmq[${PYTHON_USEDEP}]
+		>=dev-python/jupyter_client-5.3.4[${PYTHON_USEDEP}]
+		dev-python/cloudpickle[${PYTHON_USEDEP}]"
```

Comment:
Please sort alphabetically and check whether RDEPEND or DEPEND is correct.

https://github.com/spyder-ide/spyder-kernels/blob/master/ci/runtests.sh